IdentityCore/src/requests/sdk/msal/MSIDDefaultBrokerResponseHandler.m

```diff
@@ -245,6 +250,24 @@ - (NSError *)resultFromBrokerErrorResponse:(MSIDAADV2BrokerResponse *)errorRespo
     
     userInfo[MSIDBrokerVersionKey] = errorResponse.brokerAppVer;
     
+    // optional: additional_tokens
+    if (tokenResult)
+    {
+        userInfo[MSIDInvalidTokenResultKey] = tokenResult;
+    }
+    
+    // optional: MSIDGrantedScopesKey
```

Comment:
Change looks good, one thought : when we are returning the error User Info, should we return the granted scope info as well or just the decline scopes and InvalidTokenResult ?

Reply:
We are returning all the granted scopes, declined scopes and InvalidTokenResult

Reply:
sorry i meant to ask should we avoid sending the granted scope as that doesn't fall under the error category

Reply:
When there is no authenticator app installed it is already including the granted scopes, so I'm matching the same behavior when the broker app is installed.

Additionally, having the granted scopes will also match what is suggested in this document [Handle errors and exceptions in MSAL for iOS/macOS - Microsoft Entra | Microsoft Docs](https://docs.microsoft.com/en-us/azure/active-directory/develop/msal-error-handling-ios#error-handling-in-msal-for-iosmacos)

Reply:
Got it , thanks Juan, makes sense.